reactor-netty-core/src/main/java/reactor/netty/util/MapUtils.java

```diff
@@ -0,0 +1,31 @@
+package reactor.netty.util;
```

Comment:
Please add `copyright` statement.

Move this utility class in `reactor.netty.internal.util`